flow/config.go

```diff
@@ -146,15 +242,20 @@ func LoadConfig() *Config {
 
 	d := json.NewDecoder(f)
 
-	var conf Config
+	conf := new(Config)
 
-	if err := d.Decode(&conf); err != nil {
+	if err := d.Decode(conf); err != nil {
```

Comment:
Should be passed by pointer here, no? Otherwise decode won't be able to set values?

Reply:
conf is now a pointer, defined two lines above, with `new(Config)`

Reply:
Ah, missed that! 👍